src/cascadia/TerminalApp/WslDistroGenerator.cpp

```diff
@@ -13,6 +13,8 @@
 #include <fcntl.h>
 #include "DefaultProfileUtils.h"
 
+static constexpr std::wstring_view DockerDistributionPrefix{ L"docker-desktop" };

```

Comment:
This doesn't change per locale/language, does it?

Reply:
I do not believe so. They're just utility distributions created by docker for docker's use -- it would be unusual for them to localize.

Reply:
OK. Just double checking.